core/src/test/java/org/apache/iceberg/TestSnapshotJson.java

```diff
@@ -37,13 +34,16 @@ public class TestSnapshotJson {
 
   @Test
   public void testJsonConversion() throws IOException {
-    int snapshotId = 23;
-    Long parentId = null;
-    String manifestList = createManifestListWithManifestFiles(snapshotId, parentId);
-
     Snapshot expected =
-        new BaseSnapshot(
-            0, snapshotId, parentId, System.currentTimeMillis(), null, null, 1, manifestList);
+        MetadataTestUtils.buildTestSnapshotWithExampleValues()
+            .setOperation(null)
```

Comment:
I don't think operation, summary or schema ID really need to be specified here right?